src/AppInstallerCLICore/ExecutionProgress.cpp

```diff
@@ -151,6 +151,24 @@ namespace AppInstaller::CLI::Execution
                 LOG_HR(E_UNEXPECTED);
             }
         }
+
+        void ProgressVisualizerBase::ClearLine()
+        {
+            if (UseVT())
+            {
+                m_out << TextModification::EraseLineEntirely << '\r';
+            }
+            else
+            {
+                // Best effort when no VT (arbitrary number of spaces that seems to work)
+                m_out << "\r                                                              \r";

```

Comment:
Nit: How about something like `m_out << '\r' << std::string(N, ' ') << '\r';` so that we know how many spaces there are and can extend it if needed?

Reply:
Changed it to get the current console width.